go/rows.go

```diff
@@ -175,8 +194,8 @@ func (r *Rows) openResults() error {
 		_, err = r.mgr.DownloadWithContext(r.ctx,
 			scratchFile,
 			&s3.GetObjectInput{
-				Bucket: aws.String(r.config.GetOutputBucketname()),
-				Key:    aws.String(r.config.GetQueryResultKey(r.queryID)),
+				Bucket: aws.String(resultLocation.Host),
+				Key:    aws.String(strings.TrimPrefix(resultLocation.Path, "/")),
```

Comment:
jw, what's the difference between the new and old versions of these statements?

Reply:
The old version uses where we _told_ Athena to write the results; the new version uses where Athena _did_ write the results. AFAICT these will almost always be the same, but not always.